crawl-ref/docs/develop/release/pbuilderrc

```diff
@@ -142,7 +142,7 @@ export SHELL=/bin/bash
 DEBOOTSTRAP="debootstrap"
 
 # default file extension for pkgname-logfile
-PKGNAME_LOGFILE_EXTENTION="_$(dpkg --print-architecture).build"
+PKGNAME_LOGFILE_EXTENSION="_$(dpkg --print-architecture).build"
```

Comment:
Annoyingly, this change is a bug. `PKGNAME_LOGFILE_EXTENTION` is the actual name, if google is to be believed.